community/i3-gaps/build

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+set -u
+
+export CHOST="${CHOST:-x86_64-pc-linux-musl}"
+
+patch -p1 < i3-4.16-musl-GLOB_TILDE.patch
+
+./configure \
+    --prefix=/usr
+
+cd "${CHOST}" && make
```

Comment:
There's no need for `&&` as `-e` (`errexit`) is used. Move `make` on to its own line below.